quick-github-guide.md

```diff
@@ -0,0 +1,163 @@
+# Quick guide to working with a GitHub repo
+
+This page is intended for people who want to update the
+[Kubeflow docs](https://www.kubeflow.org/docs/) and
+who don't use git or GitHub often. The page gives you a quick guide to
+get going with a GitHub repo, using either the GitHub web user interface (UI)
+or git on the command line.
+
+## Using the GitHub web UI
+
+**Note:** The GitHub web UI is suitable for quick updates to a single file. If
+your update is more complex or you need to update more than one file within one
+pull request (PR), then the command line provides a better experience.
+
+Follow these steps to edit a page using the GitHub UI:
+
+1. Sign in to GitHub if you haven't yet done so.
+
+1. Go to the page that you want to edit on the
+  [Kubeflow website](https://www.kubeflow.org/docs/).
+
+1. Click **Edit this page**.
+
+1. If this is the first time you're updating a file in the Kubeflow
+  website repository, a screen opens asking you to *fork* the repository. A
+  fork is a copy of the repository where you can make your updates before
+  submitting them for review. You only have to do this once:
+
+    * Click **Fork this repository**.
+    * If GitHub asks you **Where should we fork website** and offers your
+      username as an option, click the link on your username.	
+    * Wait a few seconds while GitHub makes a copy of the repository at	
+     `https://github.com/yourusername/website`. This copy is your *fork*	
+     of the `kubeflow/website` repository.
+
+1. The GitHub editor interface opens for the selected page.
+  Make your updates to the content.
+
+1. Click **Preview changes** at the top of the editing area to see the effect of your changes.
```

Comment:
Suggestion: mention the goal before the instruction

To see the effect of your changes, click **Preview changes** at the top of the editing area.

Reply:
Actually, I prefer the pattern of telling folks what to do, and adding the explanation later. It takes less mental parsing, especially in a list of step-by-step instructions.